### ISSUE TITLE
Append patch File menu option

### ISSUE THREAD

**Issue by SteveRussell33** [VCVRack/Rack#1333]
### Summary

Rack should have a File menu option to Append a patch to the currently loaded one.
This has been an option in a Rack v0.6.2 fork previously.
https://github.com/Rcomian/Rack/commit/343c638edbf4054254785e3658e1203dc0b09097

### Motivation

[From Rcomian/Rack/Readme]
>Append an existing patch below your current patch.

>Lets you build a library of components and compose them together.

>Use the folder with a plus icon to access the append menu. You can append a whole patch, or skip the first row of the patch you're appending.

>The skip first row feature lets you build a component with audio and test trigger modules in the first row that don't get added to the combined patch when you append it.

**Comment by thouldcroft** [comment]
Yes please. For clarity and ease on my cpu I will build a experimental/test patches outside of my main performance patch. I finally get the parameters where I want them, and I realized then that I couldn't paste the experimental patch into my main patch (I come from the Max world where I do this regularly). This would also help in modularity of programming, as pre-built module chains could be re-used.
